### ISSUE TITLE
Why you not use Equalized Linear or Conv layer

### ISSUE THREAD

**Issue by Johnson-yue** [lucidrains/stylegan2-pytorch#64]
stylegan1 and stylegan2 also use it
but you code similar not use it ，why

**Comment by lucidrains** [comment]
@Johnson-yue it's meant to capture the big ideas in the paper, and not all the exact details

**Comment by lucidrains** [comment]
@Johnson-yue can you point me to where in the paper it talked about "equalized" layers?

**Comment by Johnson-yue** [comment]
In paper <Analyzing and Improving the Image Quality of StyleGAN>
![image](https://user-images.githubusercontent.com/10268274/82774910-b2b9b300-9e78-11ea-8837-3decedd9bb3e.png)

NVlabs/stylegan2
https://github.com/NVlabs/stylegan2/blob/cec605e0834de5404d5c7e5cead7053bdd0e4dde/training/networks_stylegan2.py#L22-L36

this `runtime_coef` is  equalized learning rate trick

every conv layer run `get_weight()` function first to  compute feature

**Comment by lucidrains** [comment]
@Johnson-yue I haven't found that to be too important, and none of the other researchers I'm in touch with are particularly latched onto this detail

**Comment by Johnson-yue** [comment]
as I know ,  
1.stylegan-v1
2. stylegan-v2
3. [MSG-GAN: Multi-Scale Gradient GAN for Stable Image Synthesis](https://github.com/akanimax/BMSG-GAN)
 used it .

but maybe it is not too important

**Comment by lucidrains** [comment]
@Johnson-yue you are free to submit a PR, with experimental evidence
